Update run of show with job postings (#796)
writethedocs/www: docs/conf/portland/2019/sponsors/prospectus.rst

You should have recieved a unique URL with a discount code for your free sponsorship tickets. We are happy to send it over again, just ask!

How do I use my job postings?
-----------------------------

You can post your jobs to our `job board <https://jobs.writethedocs.org/>`_.
You will be given a discount code that will let you post them for free,
please ask us for this if you don't have it! 

> They will be syncidated to our :doc:`Newsletter </newsletter>` once a month,

Reviewer: ```suggestion
They will be published in our :doc:`Newsletter </newsletter>` every month,
```

> along with listing on our website in the sidebar.

Reviewer: ```suggestion
and displayed on our website as well.
```
